R/article.R

```diff
@@ -444,3 +444,13 @@ trb_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
     "trb", keep_tex = keep_tex, citation_package = citation_package, ...
   )
 }
+
+#' @section \code{wellcomeor_article}: Format for creating submissions to
+#' Wellcome Open Research. Adapted from \url{https://www.overleaf.com/project/6131911644b635ad3aaa4cb2}.
```

Comment:
I did not checked previously but this URL does not work. It is restricted. 
Is it this one ? https://www.overleaf.com/latex/templates/wellcome-open-research-article-template/hsmhhbpxvvbj

```suggestion
#' Wellcome Open Research. Adapted from <https://www.overleaf.com/project/6131911644b635ad3aaa4cb2>.
```

Reply:
Yes, thanks for catching this